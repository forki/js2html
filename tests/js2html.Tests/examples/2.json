{
    "@id": "http://ld.nice.org.uk/prov/entity#98ead3d:qualitystandards/qs18/st1/Statement.md",
    "@type": ["prov:Entity", "owl:NamedIndividual"],
    "qualitystandard:setting": "http://ld.nice.org.uk/ns/qualitystandard/setting#Primary care setting",
    "qualitystandard:targetPopulation": ["http://ld.nice.org.uk/ns/qualitystandard/agegroup#Adults", "http://ld.nice.org.uk/ns/qualitystandard/conditiondisease#Ovarian cancer"],
    "dcterms:abstract": "Women aged 50 years or over reporting one or more symptoms occurring\npersistently or frequently that suggest ovarian cancer are offered a\nCA125 test.",
    "dcterms:title": "Quality statement 1: Symptoms and CA125",
    "prov:specializationOf": "resource:FE95BF71CE72BCB37C4E2E568E5B1F8187D8B087F1CC4459DC438DCAA07BE513",
    "_id": "resource:FE95BF71CE72BCB37C4E2E568E5B1F8187D8B087F1CC4459DC438DCAA07BE513",
    "_type": "qualitystatement"
}
