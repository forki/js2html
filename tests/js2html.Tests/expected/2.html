<div>http://ld.nice.org.uk/prov/entity#98ead3d:qualitystandards/qs18/st1/Statement.md</div>
<div>resource:FE95BF71CE72BCB37C4E2E568E5B1F8187D8B087F1CC4459DC438DCAA07BE513</div>
<div>http://ld.nice.org.uk/ns/qualitystandard/setting#Primary care setting</div>
<div>Women aged 50 years or over reporting one or more symptoms occurring
persistently or frequently that suggest ovarian cancer are offered a
CA125 test.</div>
<ul>
    <li>prov:Entity</li>
    <li>owl:NamedIndividual</li>
</ul>
